debian/mariadb-server.postinst

```diff
@@ -123,12 +123,12 @@ EOF
     # perform mariadb-upgrade, (MDEV-22678).  To keep the impact minimal, we
     # skip innodb and set key-buffer-size to 0 as it isn't reused.
     if [ -f "$mysql_datadir/auto.cnf" ] &&
-    [ -f "$mysql_datadir/mysql/user.MYD" ] &&
-    ! lsof -nt "$mysql_datadir"/mysql/user.MYD > /dev/null &&
-    [ ! -f "$mysql_datadir/undo_001" ]
+       [ -f "$mysql_datadir/mysql/user.MYD" ] &&
+       ! lsof -nt "$mysql_datadir"/mysql/user.MYD > /dev/null &&
+       [ ! -f "$mysql_datadir/undo_001" ]
     then
-      echo "UPDATE mysql.user SET plugin='unix_socket' WHERE plugin='auth_socket';" |
-      mariadbd --skip-innodb --key_buffer_size=0  --default-storage-engine=MyISAM --bootstrap 2> /dev/null
+      echo "UPDATE mysql.user SET plugin='unix_socket' WHERE plugin='auth_socket';" | \
```

Comment:
The `\` should not be necessary here (but maybe it's your linter that automatically adds it). More on the linter on general comment.

Reply:
Both seems to work with BASH the same and `shfmt` say there should not be and `beautysh` does not case neither `shellcheck` so I think I'll remove them.

Reply:
Right, both seem to work:

```
± cat test.sh 
#!/bin/bash
set -e

echo test-with-backslash | \
 cat

echo test-no-backslash |
 cat

± ./test.sh 
test-with-backslash
test-no-backslash
```